k8s/nginx/README.md

```diff
@@ -16,6 +16,31 @@ This web server application is pre-configured with an SSL certificate. Please re
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## Design
+
+![Architecture diagram](resources/nginx-k8s-app-architecture.png)
+
+### Solution Information
+
+NGINX is a multi-purpose software that delivers not only the functionality of web server but also reverse proxy, load balancer, HTTP cache and mail proxy.
+
+This specific application is designed to use NGINX for serving web content.
+
+Initial configuration of application serves some default, exemplary content. This application exposes
+two endpoints: HTTP on port 80 and HTTPS on port 443. 
+
+This application uses pre-generated certificates to configure HTTPS endpoint. The validity of the certificate used for this application is 365 days. The certificate is stored as `https1.cert` secret and private key ise stored as `https1.key` secret.
+
+### Solution Configuration
+
+If users consider using this application for production purposes then they need:
+* configure their own valid SSL certificate that will be associated with appropriate DNS domain name
+* upload their web content to K8s application
+
+The instructions how to update the certificate in case NGINX application is up and running are documented in the section [Re-configure certificate of your NGINX server](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/nginx#re-configure-certificate-of-your-nginx-server)
```

Comment:
Please delete `https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/nginx` prefix and use just `#re-configure-certificate-of-your-nginx-server`.

This have three benefits:

1. If the branch will be changed, the link will be work in that branch.
1. If the repo name will be change, or it will be migrated from github, the link will be still work.
1. We should really use the link to a fragment instead of reload the page. Please read [Fragment identifier](https://en.wikipedia.org/wiki/Fragment_identifier)

Reply:
Done. Thanks for the hint.